relayer/app/cursors/cursors.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package relayer.app.cursors;
+
+import "cosmos/orm/v1/orm.proto";
+
+option go_package = "relayer/app";
+
+message Cursor {
+  option (cosmos.orm.v1.table) = {
+    id: 1;
+    primary_key: { fields: "src_chain_id,conf_level,dst_chain_id,attest_offset" }
+    index: { id: 1, fields: "confirmed" }
+    index: { id: 2, fields: "src_chain_id,conf_level,dst_chain_id" }
```

Comment:
not sure this index is required, since you have `CursorPrimaryKey{}.WithSrcChainIdConfLevelDstChainId` which is the same thing? Can remove?

Reply:
also, `confirmed` index not used (yet)